scripts/perftune.py

```diff
@@ -549,13 +563,27 @@ def _get_irqs(self):
         pass
 
 #### Private methods ############################
+    def __refresh_metadata_token(self):
```

Comment:
Another confusing function name: you are reading a token here in order to use it later - not "refreshing" it.

`__generate_metadata_token` is a more appropriate name IMO.

Reply:
Renamed this method to `__metadata_token(self)` to make it property:
https://github.com/scylladb/seastar/pull/2560#discussion_r1861306581